appengine/mailjet/app.standard.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START app_yaml]
+runtime: nodejs8
+
+env_variables:
+  MJ_APIKEY_PUBLIC: <your-mj-apikey-public>
+  MJ_APIKEY_PRIVATE: <your-mj-apikey-private>
```

Comment:
Should this not be checked into source control?

If so, perhaps we should add a comment telling people not to commit this file.

Reply:
Done :)